model/task/task.go

```diff
@@ -2597,21 +2584,7 @@ func ArchiveMany(tasks []Task) error {
 	if err != nil {
 		return errors.Wrap(err, "archiving tasks")
 	}
-
-	eventLogErrs := grip.NewBasicCatcher()
-	for _, t := range tasks {
-		if t.IsHostTask() {
-			// Host event logs involving running a host task don't include the
-			// execution number but need it to distinguish which task execution
-			// it ran once the task is no longer the latest execution. This
-			// retroactively sets the task execution for event logs involving
-			// the host running this task so that it correctly identifies this
-			// archived execution.
-			eventLogErrs.Wrapf(event.UpdateHostTaskExecutions(t.HostId, t.Id, t.Execution), "updating execution %d of task '%s' for host '%s'", t.Execution, t.Id, t.HostId)
-		}
-	}
-
-	return eventLogErrs.Resolve()
+	return nil
```

Comment:
does it make sense to return the previous `errors.Wrap(err, "archiving tasks")` now?